src/main/java/net/hypixel/nerdbot/api/database/greenlit/GreenlitMessage.java

```diff
@@ -17,11 +17,19 @@
 public class GreenlitMessage {
 
     private ObjectId id;
-    private String userId, messageId, greenlitMessageId, suggestionTitle, suggestionContent, suggestionUrl, channelGroupName;
+    private String userId;
+    private String messageId;
+    private String greenlitMessageId;
+    private String suggestionTitle;
```

Comment:
at some point it may be worth having a suggestion object

Reply:
Seems a little overkill unless we're gonna use it somewhere else too